roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -276,6 +307,7 @@ spec:
               mountPath: "/var/run/redis"
             - name: rsyslog-socket
               mountPath: "/var/run/awx-rsyslog"
+          resources: {{ rsyslog_resource_requirements }}   
```

Comment:
@marekzebro I think you are going to need to move this line back down below the bundle_cacert_secret block or else there will be templating errors when bundle_cacert_secret is defined.

Reply:
You are right, this should be fixed now.